Рязанова Екатерина

anbtrfl opened the pull request:
https://codesandbox.io/p/sandbox/6kklhf

anbtrfl committed:
Add files via upload

mokhov commented:
Кажется задание не выполнено :-(
<img width="724" alt="Screenshot 2024-11-17 at 08 32 57" src="https://github.com/user-attachments/assets/ca6002f9-b509-4d18-a5d7-2bea8101d668">

mokhov commented on resume.html:
```diff
@@ -0,0 +1,464 @@
+        }
+
+        .pic-phone {
+            content: url("data:image/webp;base64,UklGRmIIAABXRUJQVlA4TFYIAAAvfwAfELfnOJIkR8neQ8MPM/DfObWNtkFyI0mOJFnUcln2tfrrxe/KHbdtJEl0zXPmtwlvkhvFJnCXYUm2rTrJeVnPkuBunwydsbnD83W5Ctu2QTrm3YEVgntc3h/sXl+sHn+Ms2QlK8vtiaioKOLONJk1UVQPxc4+MACTA3QQIay2QEbYmaDZMQFkpKcgZAOgZVESMlMCQ9iXBEUcABrAZIAAAC0CABQIrL80QIH4vPD1BwLFV4RA8er7pdcHf4n+Ml3pKEmJd0tGV01iWDCQxYAOVgPIGF0MqA2kRIsT7CcJQrorLzB0kCiaP/oLK/8lAE ...
```
Это выглядит как кликабельный элемент и должно быть ссылкой

mokhov replied:
ой ёй, мы в прошлых заданиях уже обращали на это внимание

mokhov commented on resume.html:
```diff
@@ -0,0 +1,464 @@
+        }
+
+        .pic-phone {
+            content: url("data:image/webp;base64,UklGRmIIAABXRUJQVlA4TFYIAAAvfwAfELfnOJIkR8neQ8MPM/DfObWNtkFyI0mOJFnUcln2tfrrxe/KHbdtJEl0zXPmtwlvkhvFJnCXYUm2rTrJeVnPkuBunwydsbnD83W5Ctu2QTrm3YEVgntc3h/sXl+sHn+Ms2QlK8vtiaioKOLONJk1UVQPxc4+MACTA3QQIay2QEbYmaDZMQFkpKcgZAOgZVESMlMCQ9iXBEUcABrAZIAAAC0CABQIrL80QIH4vPD1BwLFV4RA8er7pdcHf4n+Ml3pKEmJd0tGV01iWDCQxYAOVgPIGF0MqA2kRIsT7CcJQrorLzB0kCiaP/oLK/8lAE ...
```
Идеал — сделать это списком

anbtrfl replied:
не очень поняла,что вы имеете в виду
списком это как ?

anbtrfl committed:
Update resume.html

исправила замечания, тест accessibility сейчас проходит 92% (скрин приложу ) с жалобой на frame/iframe. я дописала скрипт, который должен это исправлять, но почему-то не работает ,я не понимаю, что не так

anbtrfl committed:
Add files via upload

mokhov commented:
Основная часть задания всё ещё не выполнена :(

mokhov commented:
Зачли на паре
